File: Units/parser-perl.r/skip-heredoc.d/input-0.pl
# Derived from https://github.com/geany/geany/blob/master/scripts/fix-alignment.pl
(($argc == (1 << 0x1)) or ($argc >= (1<<1) and $opt_write)) or die <<END;
Usage:
$scriptname sourcefile [>outfile]
  Print formatted output to STDOUT or outfile.
  Warning: do not use the same file for outfile.
$scriptname -w sourcefile(s)
  Writes to the file(s) in-place.
  Warning: backup your file(s) first or use clean version control files.
END

sub foo {
    print "hello\n";
}

File: Units/parser-ruby.r/ruby-alias.d/input-0.rb
# Derived from https://github.com/tmm1/ripper-tags/blob/master/test/test_ripper_tags.rb
module M
  def get(); end
  def set(); end

  alias :"[]" :get
  alias :"[]=" :set
  alias :set :"[]="
end

def z(); end
